tox.ini

```diff
@@ -0,0 +1,53 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = true
+envlist =
+    py35-sqlite
+    flake8
+    docs
+    isort
+
```

Comment:
I would add an extra `skip_missing_interpreters = true` as well - for example on my system I have Python 2.7 and Python 3.5 by default, but no Python 3.4. Others may have only Py2 or Py3. They can then still easily run the build with `tox` without getting nasty errors and the 'burden' to install an (older) version of Python that's might not be in their package managers.